third_party/libunwindstack/DwarfEncoding.h

```diff
@@ -46,6 +46,27 @@ enum DwarfEncoding : uint8_t {
   DW_EH_PE_block = 0x0f,
 };
 
+// Needed for static_asserts that depend on the template parameter. Just a
+// plain `false` in the static_assert will cause compilation to fail.
+template <class>
```

Comment:
nit: I'd use `typename `for consistency.

Reply:
Done.